### PR TITLE
chore(project): update format:diff ignore pattern

### PR DESCRIPTION
**Pull request by joshblack**
Updates our `format:diff` to include `react`. The `components` pattern was mistakenly stopping this from occurring. Also updates failures from this change.

#### Changelog

**New**

**Changed**

- `format:diff` now uses a separate ignore pattern with `packages/components` instead of `components`
- Updated files according to prettier formatting

**Removed**

### CODE REVIEW

**Comment by joshblack** [comment]
FYI @abbeyhrt, I think you might have been running into this

**Comment by abbeyhrt** [comment]
Oh nice, thanks for looking into this! Should I wait for the to be merged before merging the endgame guide update?

**Comment by joshblack** [comment]
@abbeyhrt I'm up for whatever ends up getting merged in first 👍 No harm either way

**Review by abbeyhrt — APPROVED** [review]

**Pull request merged**
